test/com/facebook/buck/cli/DaemonIntegrationTest.java

```diff
@@ -93,7 +93,7 @@ public void whenConcurrentCommandExecutedThenSecondCommandFails()
       throws IOException, InterruptedException, ExecutionException {
 
     ProjectWorkspace workspace =
-        TestDataHelper.createProjectWorkspaceForScenario(this, "exclusive_execution", tmp);
+        TestDataHelper.createProjectWorkspaceForScenario(this, "exclusive_execution", tmp, true);
```

Comment:
I wonder if we can make it default.

Reply:
I don't see why not, I can clean this up (since I've already started)